src/Reveal.Sdk.Dom.Tests/Visualizations/GridVisualizationFixture.cs

```diff
@@ -0,0 +1,92 @@
+﻿using Reveal.Sdk.Dom.Data;
+using Reveal.Sdk.Dom.Visualizations;
+using Reveal.Sdk.Dom.Visualizations.Settings;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Xunit;
+
+namespace Reveal.Sdk.Dom.Tests.Visualizations
+{
+
+    public class GridVisualizationTests
```

Comment:
It seems that we are missing test for ` public GridVisualization(DataSourceItem dataSourceItem) : this(null, dataSourceItem) { }`

Reply:
Also check for the naming convensions. This should be `GridVisualizationFixture`